<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnswerListItem</class>
 <widget class="QWidget" name="AnswerListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>37</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resource/head_pic1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_answer_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-3</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0%);</string>
   </property>
   <property name="text">
    <string>JackChance</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_answer_detail">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>42</y>
     <width>651</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255,0%);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;就是副卡世界风你啊JFK老师解答就是考虑到房价啊额就开始了解flak艰苦拉萨解放了卡就利卡离开房间啊垃圾分厘卡撒酒疯离开撒看风景阿里山&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_answer_time">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0%);</string>
   </property>
   <property name="text">
    <string>周三 09 14 2017</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_diss">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>96</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resource/diss_unclicked.png);
background-color: rgb(255, 255, 255,0%);
border-radius:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_praise">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>92</y>
     <width>30</width>
     <height>24</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0%);
border-radius:0px;
border-image: url(:/Resource/praise_unclick.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label__number_praise">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>92</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0%);</string>
   </property>
   <property name="text">
    <string>25</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_diss_number">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>92</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0%);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
